src/components/profile/likehistory.js

```diff
@@ -1,12 +1,123 @@
-import { Box, Typography } from "@mui/material";
-import React from "react";
+// import { Box, Typography } from "@mui/material";
+// import React from "react";
+// import Tuits from "../tuits";
+//
+// const Likedhistory = () => {
+//
+//   return (
+//     <Box>
+//       <Typography variant="h3">Likedhistory</Typography>
+//
+//     </Box>
+//   );
+// };
+//
+// export default Likedhistory;
 
+import { Box, Stack, Typography } from "@mui/material";
+import React, { useEffect, useState } from "react";
+
+import Tuits from "../tuits";
+import tuitsArray from "../tuits/tuits-data.json";
+import Bio from "./bio";
+
+import * as authService from "../../services/auth-service";
+import * as tuitService from "../../services/tuits-service";
+import * as serviceProfile from "../../services/profiles-service";
+import * as serviceUser from "../../services/users-service";
+import Skeleton from "@mui/material/Skeleton";
+import { useNavigate } from "react-router-dom";
+
+const Placeholder = (
+    <Stack spacing={1}>
+        {/* For variant="text", adjust the height via font-size */}
+        <Skeleton variant="text" sx={{ fontSize: "1rem" }} />
+
+        {/* For other variants, adjust the size with `width` and `height` */}
+        <Skeleton variant="circular" width={40} height={40} />
+        <Skeleton variant="rectangular" width={210} height={60} />
+        <Skeleton variant="rounded" width={210} height={60} />
+    </Stack>
+);
+
+/**
+ * Profile component with Bio and Tuit component
+ */
 const Likedhistory = () => {
-  return (
-    <Box>
-      <Typography variant="h3">Likedhistory</Typography>
-    </Box>
-  );
-};
+    const navigate = useNavigate();
+    const [authprofile, setAuthProfile] = useState({});
+    const [profileResp, setProfile] = useState({});
+    const [userResp, setUser] = useState({});
+    // const [myTuits, setMyTuits] = useState();
```

Comment:
setMyTuits is commented but being used on line 83

Reply:
I think we can sequence the 3 effects into a small one

Reply:
Please don't merge your code to main. Instead merge to spring5-integration-react

Reply:
got it no worry I see this